pyproject.toml

```diff
@@ -166,7 +173,12 @@ ignore = [
     "A001",  # Variable `copyright` is shadowing a Python builtin
     "RUF003",  # Comment contains ambiguous {}
     "RUF019",  # [*] Unnecessary key check before dictionary access
+    "E266",  # Too many leading `#` before block comment
```

Comment:
These ignores are only for `docs/conf.py`. Some minor changes are done in that file so that we don't have to put a whole lot more exceptions here.